JavaScript/7-single.js

```diff
@@ -0,0 +1,64 @@
+
+const Node = (value, next = null) => ({
```

Comment:
It is not a class or prototype constructor so I'd like to use lowerCamelCase here.